remoteappmanager/handlers/home_handler.py

```diff
@@ -90,9 +90,9 @@ def _actionhandler_start(self, options):
         _, app, policy = choice[0]
 
         container = None
-        orm_user = self.current_user.orm_user
+        user_name = self.current_user.name
         try:
-            container = yield self._start_container(orm_user,
+            container = yield self._start_container(user_name,
```

Comment:
We have agreed that the singleuserapp's user holds an opaque type `account`, we can no longer make assumption on what it contains.